main.go

```diff
@@ -124,6 +124,14 @@ func getAgentConfigPathFromHome() string {
 	return path.Join(home, "wiresteward.json")
 }
 
+func getAgentTokFilePathFromHome() string {
```

Comment:
We've had issues with HOME before, might want to have a look at: https://github.com/uw-labs/strongbox/blob/master/strongbox.go#L115

Reply:
I've had troubles with that in the past in darwin, I've successfully used https://github.com/mitchellh/go-homedir